extensions/amp-form/0.1/amp-form.css

```diff
@@ -18,3 +18,28 @@ form.amp-form-submit-success [submit-success],
 form.amp-form-submit-error [submit-error] {
     display: block;
 }
+
+
+.-amp-validation-bubble {
+    background-color: white;
+    box-shadow: 0 5px 15px 0 rgba(0, 0, 0, .5);
+    max-width: 200px;
+    position: absolute;
+    top: -999px;
```

Comment:
Using [transforms](https://github.com/ampproject/amphtml/pull/3734#discussion_r68319778), we can use `display: none`.